src/ontology/mondo-edit.obo

```diff
@@ -457367,6 +457355,21 @@ is_a: MONDO:0019118 {source="https://clinicalgenome.org/affiliation/40072/"} ! i
 relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/2150 {source="https://clinicalgenome.org/affiliation/40072/"} ! CNGA3
 property_value: IAO:0000233 https://github.com/monarch-initiative/mondo/issues/4937 xsd:anyURI
 
+[Term]
+id: MONDO:0800103
+name: COACH syndrome 1
+def: "Any COACH syndrome in which the cause of the disease is a variation in the TMEM67 gene." [MONDO:patterns/OMIM_disease_series_by_gene]
+synonym: "cerebellar vermis hypo/aplasia, oligophrenia, congenital ataxia, ocular coloboma, and hepatic fibrosis" EXACT [OMIM:216360]
+xref: ICD10CM:Q04.3 {source="Orphanet:1454/attributed", source="Orphanet:1454/ntbt", source="Orphanet:1454"}
+xref: OMIM:216360 {source="Orphanet:1454", source="MONDO:equivalentTo", source="Orphanet:1454/e"}
+is_a: MONDO:0015369 {source="Orphanet:1454"} ! Joubert syndrome and related disorders
+is_a: MONDO:0015508 {source="MONDO:Redundant", source="Orphanet:1454"} ! genetic parenchymatous liver disease
+is_a: MONDO:0021147 {source="MONDO:Entailed", source="Orphanet:1454", source="Orphanet:1454/inferred"} ! disorder of development or morphogenesis
```

Comment:
this also seems like it should be a superclass on the parent class (COACH syndrome)? What do you think?